src/axiosConfig.js

```diff
@@ -6,7 +6,6 @@ const axiosConfig = axios.create({
   baseURL: REACT_APP_BASE_URL,
   headers: {
     'content-type': 'application/json',
```

Comment:
In case this is merged before the axios bug fix, it will erase the axios config which I need to implement some of my features